demisto_sdk/commands/test_content/TestContentClasses.py

```diff
@@ -1501,20 +1515,13 @@ def _execute_mockable_test(self, proxy: MITMProxy):
                 return False
 
         # Running playback after successful record to verify the record is valid for future runs
-        if status == PB_Status.COMPLETED:
+        if succeed:
             self.build_context.logging_module.info(
                 f'------ Test {self} start ------ (Mock: Second playback)')
             with run_with_mock(proxy, self.playbook.configuration.playbook_id) as result_holder:
                 status = self._run_incident_test()
                 result_holder[RESULT] = status == PB_Status.COMPLETED
-                self._clean_incident_if_successful(status)
```

Comment:
revert this

Reply:
Moved it into the `_run_incident_test` method